demo-app/src/app/navigation/navigation.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="container" fxLayout="row" fxLayoutAlign="space-between center">
```

Comment:
![navigation](https://user-images.githubusercontent.com/105719212/170632301-ea030e28-78c4-44f4-b55a-2d598cd5a826.png)

Reply:
make text and icon same alignment